baselines/polars/tpch/run

```diff
@@ -0,0 +1,11 @@
+export LOG_TIMINGS=1
```

Comment:
You want to remove this file from the commit since we have `run.sh`.

Reply:
I delete this file in the new push